vector/src/main/java/im/vector/app/features/roomprofile/settings/RoomSettingsController.kt

```diff
@@ -144,5 +148,53 @@ class RoomSettingsController @Inject constructor(
                 id("guestAccessDivider")
             }
         }
+
+        // Security
+        buildProfileSection(stringProvider.getString(R.string.room_profile_section_security))
+
+        data.globalCryptoConfig.invoke()?.let { globalConfig ->
+            if (globalConfig.globalBlacklistUnverifiedDevices) {
+                genericFooterItem {
+                    id("globalConfig")
+                    centered(false)
+                    text(
+                            span {
+                                +host.stringProvider.getString(R.string.room_settings_global_blacklist_unverified_info_text)
+                                apply {
+                                    if (data.unverifiedDevicesInTheRoom.invoke() == true) {
+                                        +"\n"
+                                        +host.stringProvider.getString(R.string.some_devices_will_not_be_able_to_decrypt)
+                                    }
+                                }
+                            }.toEpoxyCharSequence()
+                    )
+                    itemClickAction {
```

Comment:
maybe you don't have to add this parameter :)

Reply:
good catch. Added the option to open global settings to update the value of block/unblock